conf/test_ancient_dna.config

```diff
@@ -33,7 +33,7 @@ params {
     skip_krona                           = true
     min_length_unbinned_contigs          = 1
     max_unbinned_contigs                 = 2
-    busco_db                             = "https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2024-01-08.tar.gz"
+    busco_db                             = "https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb12.2024-11-14.tar.gz"
```

Comment:
Does this still work with odb10 lineages?

Reply:
(thinking about @jfy133's busco lineage catch PR)

Reply:
Validated that this version of BUSCO works with both odb10 and odb12 lineages - so we can merge this and ignore @jfy133's PR